src/clients/node/src/node.zig

```diff
@@ -372,7 +372,8 @@ fn decode_array(comptime Event: type, env: c.napi_env, array: c.napi_value, even
                     const value = try @field(translate, @typeName(FieldInt) ++ "_from_object")(
                         env,
                         object,
-                        @ptrCast(field.name ++ "\x00"),
+                        // Use `comptimePrint` to ensure the sentinel terminated string.
+                        std.fmt.comptimePrint("{s}", .{field.name}),
```

Comment:
It's not clear if `field.name` already contains a sentinel, since it's a `comptime` string, it should, but just removing the `++ "\x00"` didn't work.

Reply:
As an alternative, consider defining a helper

```zig
fn add_trailing_nul(comptime s: []const u8) [:0] const u8 {
    const result = s ++ "";
    assert(result.len == s.len);
    assert(result[result.len] = 0;
    return result;
}
```

That `++ ""` looks mystifying, but I think it's simpler than `comptimePrint`, and the signature & asserts hopefully make it clear what's going on here!

Reply:
Yeah, we just added two sentinels. Comptime concat ++ knows how to deal with sentinels when adding two [:0], but in our case the sentinel was part of the string!

I didn't like ++ "" very much, as you said, too mystical. But yes, with a good comment it seems reasonable.

Reply:
>I didn't like ++ "" very much, as you said, too mystical

My thoughts exactly! But then I figured that I don't actually understand why `comptimePrint` works here any more than I do with `++ ""` --- it is equally mystic, but also has enough stuff going on to serve as a [thought terminator](https://en.wikipedia.org/wiki/Thought-terminating_cliché).